src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -3085,7 +3085,7 @@ class C { }
             // NOTE: adding new output types will cause this test to fail. Update above as needed.
             foreach (IncrementalGeneratorOutputKind kind in Enum.GetValues(typeof(IncrementalGeneratorOutputKind)))
             {
-                if (kind == IncrementalGeneratorOutputKind.None)
+                if (kind == IncrementalGeneratorOutputKind.None || kind == IncrementalGeneratorOutputKind.Host)
```

Comment:
Instead of skipping this output kind should we update the test as the comment says

```cs
// NOTE: adding new output types will cause this test to fail. Update above as needed.
```

Wouldn't that be just matter of adding a `ctx.RegisterHostOutput` line?

Reply:
Yeah, I looked at doing that, but the rest of the test is checking against the `GeneratedSources` collection, because up to now they all added to that. It didn't seem worth updating it to special case it, but its simple enough I can.